packages/core-data/src/resolvers.js

```diff
@@ -661,12 +661,42 @@ export const getUserPatternCategories =
 
 export const getNavigationFallbackId =
 	() =>
-	async ( { dispatch } ) => {
+	async ( { dispatch, select } ) => {
 		const fallback = await apiFetch( {
-			path: addQueryArgs( '/wp-block-editor/v1/navigation-fallback' ),
+			path: addQueryArgs( '/wp-block-editor/v1/navigation-fallback', {
+				_embed: true,
+			} ),
 		} );
 
+		const record = fallback?._embedded?.self;
+
 		dispatch.receiveNavigationFallbackId( fallback?.id );
+
+		if ( record ) {
```

Comment:
I know this is specific to the previous regression, but should we check for `content.raw` existence in the `record` before seeding it to the cache?